io/module_write_internal_state.F90

```diff
@@ -1,16 +1,10 @@
-!-----------------------------------------------------------------------
-      module write_internal_state
-!
-!-----------------------------------------------------------------------
-!***  the internal state of the write component.
-!-----------------------------------------------------------------------
-!***
-!***  revision history
-!***
-!       Feb 2017:  J. Wang - Initial code
-!
-!-----------------------------------------------------------------------
-!
+!> @file
+!> @brief The internal state of the write component.
+!> @author Jun Wang @date Feb, 2017
+
+!> @brief The internal state of the write component.
+!> @author Jun Wang @date Feb, 2017
```

Comment:
Why is this repeated twice?

Reply:
Once for the file, once for the module. Doxygen needs both.

Reply:
Ok. Thanks.